org.americanmonk.TipitakaPaliReader.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.americanmonk.TipitakaPaliReader</id>
+  
+  <name>Tipitaka Pali Reader</name>
+  <summary>This is an app to read the Theravāda Buddhist Tipitaka Pāḷi texts.  It allows one to click on words and get definitions effortlessly.  </summary>
```

Comment:
Ideally your summary should be shorter. This is not a requirement, but just a tip.

It also shouldn't end with a fullstop or spaces :)

Reply:
I can change it.. but is that why my PR is not approved?  I'll have to figure out how to do a pr update.  But as you said.. not required.

Reply:
I should mention that the xml was just checked into the root to satisfy some type of requirement.  It is actually bundled into the gz located in the binary source download.  I highly doubt that changing the xml that I submitted to satisfy this requirement will actually affect anything.  I'm not sure how to do packaging with an external desktop and xml.  Is there an example on how to do that.. like someone who was accepted and did a binary submission?  TBH.. the bundling of all the stuff in a single directory threw me off for some time.  I'd also like to do that without the directories but I don't have any sample files.  If I had a sample.. i could duplicate it.  Please include a sample.

for the mean time.. i'll push an edit to the root xml in this PR, but not in the gz binary bundle.